Client/Frontend/Home/HomePanelViewController.swift

```diff
@@ -71,8 +72,10 @@ class HomePanelViewController: UIViewController, UITextFieldDelegate, HomePanelD
         controllerContainerView = UIView()
         view.addSubview(controllerContainerView)
 
-        blur.snp_makeConstraints { make in
-            make.edges.equalTo(self.view)
+        if DeviceInfo.isBlurSupported() {
```

Comment:
Same here plz -- if blur isn't supported, don't set `blur` in the first place (or add it as a subview).